.circleci/config.yml

```diff
@@ -49,15 +49,15 @@ workflows:
           filters:
             tags:
               only:
-                - /^(.*)$/
+                - /^(\d.*)$/
```

Comment:
we can build only `v` tags here, would it work? 
i.e. we do it in the [gdal-warp-bindings](https://github.com/geotrellis/gdal-warp-bindings/blob/master/.circleci/config.yml#L120)

Reply:
That's a good thing to try.

I think the difference here is that we rely on `chan` for changelog edits and rotation / github release creation. I'm not sure if the issue is just that it's not signing tags, or if `chan` and `sbt-git` [disagree](https://github.com/azavea/stac4s/blob/master/build.sbt#L8-L13) about how to figure out the version, or what. We can science this though -- for the client signatures release, I'll tag HEAD with a `v0.0.21` before doing `chan gh-release` and see if I run into any conflicts.

Reply:
@jisantuc :+1: